study.md

```diff
@@ -54,5 +56,5 @@ like enumerables?  How are they different?  Please compare and contrast these
 types.
 
 ```md
-<!-- your answer here -->
+Streams are similar to Enumerables because in both instances items in enumerables when interated over and items in a stream are processed one at a time. They are different because hashes, ranges, and arrays are finite whereas Streams are potentially infinite. 
```

Comment:
Streams should generally be treated as infinite, even though they can be finite, and enumerables should generally be treated as finite, even though they can be infinite.